### PR TITLE
lint from root

### PR DESCRIPTION
**Pull request by chicoxyzzy**
this completes #3123

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -7,5 +7,10 @@ src/**/__tests__/**
 # This should be enabled but that folder has too much in it that doesn't belong
 src/test
 test/the-files-to-test.generated.js
-# This is synced with a different file internally, don't want to lint it yet
-vendor/fbtransform/syntax.js
+vendor/
```
Comment (chicoxyzzy) [inline .eslintignore]: Not sure about comment that was here before. Grunt task didn't lint `vendor/` before but comment was about `vendor/fbtransform/syntax.js` only.

Comment (zpao) [inline .eslintignore]: It does as of #3206

Comment (chicoxyzzy) [inline .eslintignore]: should this be
```
# This is synced with a different file internally, don't want to lint it yet
vendor/fbtransform/syntax.js
vendor/jasmine/
vendor/jasmine-jsreporter/
vendor/*.js
```
then?

Comment (chicoxyzzy) [inline .eslintignore]: done

**Comment by jsfb** [comment]
cc @zpao

**Comment by zpao** [comment]
Thanks!

**Pull request merged**